corehq/apps/locations/views.py

```diff
@@ -212,6 +212,9 @@ def _get_loc_types(self):
             'expand_from': loc_type.expand_from.pk if loc_type.expand_from else None,
             'expand_from_root': loc_type.expand_from_root,
             'expand_to': loc_type.expand_to.pk if loc_type.expand_to else None,
+            'include_without_expanding': (loc_type.include_without_expanding.pk
+                                          if loc_type.include_without_expanding else None),
```

Comment:
Yeah, it's how django does foreign key references.  When you define a foreign key field (say, `SQLLocation.parent`), Django actually stores the related object's `id` (or `pk` - they're synonyms) as `<field_name>_id`.

* `location.parent.id` looks up the parent and returns the `pk` of the doc, just like how `location.parent.name` would work
* `location.parent_id` *is* the reference to the parent id, so it doesn't need to look up the parent document in the db - it's stored on the `SQLLocation` model.

Aside:
A while back, I wanted to make a `SQLLocation.parent_id` method to mirror the one on `Location`, but django has that attribute set already, so it breaks.  Unfortunately, `SQLLocation.id != SQLLocation.location_id`, so we can't just use that, and we *do* need to do the lookup every time (prefetch_related would help though).  For `LocationType`, though, there's only one id, so it's simpler.